home/index.php

```diff
@@ -107,8 +107,20 @@
                     echo "<li>" .
                         "<h3>" . $hackathon["theme"] . "</h3>" .
                         "<p>" . $hackathon["description"] . "</p>" .
-                        "<p>" . $hackathon["dateStart"] . " to " . $hackathon["dateEnd"] . "</p>" .
-                        "</li>";
+                        "<p>" . $hackathon["dateStart"] . " to " . $hackathon["dateEnd"] . "</p>";
+
+                    $winner_sql = "SELECT title, username FROM projects NATURAL JOIN users
```

Comment:
use inner join tapos on projecs.winning = users.projectid

Reply:
whats projects.winning?

Reply:
wala ring projectID column sa users

Reply:
AY JOKE

dapat triple join pala to


(hackathons inner join projects on hackathons.winningProjectID == projects.projectId) inner join users using userID